src/api/java/com/minecolonies/api/compatibility/Compatibility.java

```diff
@@ -29,6 +32,8 @@ private Compatibility()
         throw new IllegalAccessError("Utility class");
     }
 
+    public static IBeehiveCompat beeHiveCompat = null;
```

Comment:
Can't we give this the default value already?

Reply:
(javadoc please too)

Reply:
ye could do = new IBeehiveCompat(){}

Reply:
After this small adjustment its ready to merge.

Reply:
@truenachtara